CHANGELOG.md

```diff
@@ -4,6 +4,10 @@
 
 Not yet released; provisionally v2.0.0 (may change).
 
+### New Docker ARGS: GOFLAGS and GO111MODULE 
+The Trillian Docker images now accept GOFLAGS and GO111MODULE arguments
+and set them as environment variables inside the Docker container.
```

Comment:
It might be best to put this in the "Deployments" section, since that already has lots of information about changes to Docker-related stuff.